test/persona_client_tests.js

```diff
@@ -145,18 +145,6 @@ describe("Persona Client Test Suite - Constructor & Token Tests", function() {
                     });
                 });
             });
-            it("should not return a token",function(done) {
-                var personaClient = persona.createClient("test-suite",personaClientConfig);
-
-                personaClient.obtainToken({id: "primate", secret: "wrong_password"},function(err, data) {

```

Comment:
This test makes no sense. It suggests that using the wrong password should not return a cached token. As the user already has access to the cache it doesn't provide any security. The key has been changed to only include the id rather than the secret. If we cannot affort to loose a secret we should not use it as part of a cache key.